pyscriptjs/src/main.ts

```diff
@@ -227,6 +231,8 @@ from pyscript import micropip, Element, console, document`);
         await runtime.installPackage(this.config.packages);
         await this.fetchPaths(runtime);
 
+        //This may be unnecessary - only useful if plugins try to import files fetch'd in fetchPaths()
+        runtime.invalidate_module_path_cache()
```

Comment:
I think this is unnecessary because you are already calling `runtime.invalidate_module_path_cache()` inside `fetchPythonPlugins` before each plugin is imported, so (I think!) the cache would be invalidated anyway. With that said, I think it's totally fine to keep this here anyway and remove it later